playbooks/deploy-vms.yml

```diff
@@ -183,17 +209,38 @@
         - hostvars[item]['server_vm'] | default(false) | bool
       with_items: "{{ groups['pxe_servers'] }}"
 
-    - name: Setup VirtualBMC for each VM for Undercloud to use
+    - name: Tear down existing VirtualBMC sessions
       shell: |
         vbmc stop {{ hostvars[item]['server_hostname'] }}
         vbmc delete {{ hostvars[item]['server_hostname'] }}
+      failed_when: false
+      when:
+        - hostvars[item]['server_vm'] | default(false) | bool
+      with_items: "{{ groups['pxe_servers'] }}"
+
+    - name: Get running vbmc processes
```

Comment:
i've been using `pkill -9 vbmc; rm -rf /root/.vbmc/` between `build.sh` runs.

`kill $(cat /root/.vbmc/master.pid)` sadly doesn't work, with or without `-9`/SIGKILL.

Reply:
Ah, good idea to nuke the directory too.  I had attempted to use systemd unit files but the way the daemon runs is odd and causes issues.